app/controllers/merchant/merchant_items_controller.rb

```diff
@@ -1,6 +1,25 @@
 class Merchant::MerchantItemsController < Merchant::BaseController
+  def index
+    @user = current_user
+  end
 
-    def index
-        @user = current_user  
+  def update
```

Comment:
We might be able to keep all edit function within update.
I was thinking we do an if else statement between `US42/43` & `US47&48`.
If it doesn't work, we can change it.

Reply:
That sounds good. We can work on that today.